plugins/importexport/native/templates/results.tpl

```diff
@@ -8,15 +8,22 @@
  * List of operations this plugin can perform
  *}
 
-{translate key="plugins.importexport.native.importComplete"}
-<ul>
-	{foreach from=$content item=contentItem}
-		<li>
-			{if is_a($contentItem, 'Submission')}
-				{$contentItem->getLocalizedTitle()|strip_unsafe_html}</li>
-			{else}
-				{$contentItem->getIssueIdentification()|escape}
-			{/if}
-		</li>
+{if $validationErrors}
+	<h2>Validation errors:</h2>
```

Comment:
may i put the key in the manager.xml in the pkp-lib? -- it will be then used by omp too...